source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -79,7 +79,10 @@ A **new order** webhook request is triggered upon the placement of a new order.
 ### Order updates
 
 An **order update** event is triggered upon the update of an order. As of right now, an update
-event notification is sent on order **cancellation**.
+event notification is sent for the following cases:
+
+* Order **cancellation**, triggered if an order has been cancelled by either the user or our support.
+* Order expiration time **extension**,  triggered if an order's expiration time has been extended.
```

Comment:
It's not expiration time, it's the latest time the merchant should dispatch the order.

Reply:
Well, extension action changes both timestamps so we should rephrase that in a more generic way like "the available time to handle the order is extended" or smth

Reply:
How about the following:
* Order **extension**,  triggered if the available time to handle the order is extended.

Reply:
Another option would be

```
- When an order is **cancelled** (either the user or our support team).
- When an order is **extended** (allowing more time to be handled).
```

Reply:
I agree.